src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -3371,12 +3371,6 @@ Although C# distinguishes between out and ref, the CLR sees them as the same. Wh
 
 Give the compiler some way to differentiate the methods. For example, you can give them different names or provide an additional parameter on one of them.</value>
   </data>
-  <data name="WRN_MultipleRuntimeOverrideMatches" xml:space="preserve">
```

Comment:
>WRN_MultipleRuntimeOverrideMatches [](start = 14, length = 34)

I think this warning is still applicable in some scenarios, even with explicit MethodImpl.

Reply:
Specifically when two methods collide due to, for example, generic substitution in the base type. Consider adding a test like that and make sure that changing the order of declaration for methods in the base doesn't affect what method is getting overridden. Same for the scenario in  #44067. 

---
In reply to: [441902428](https://github.com/dotnet/roslyn/pull/44025#discussion_r441902428) [](ancestors = 441902428)

Reply:
This warning was not used for that scenario. A different error (`CS0462` `ERR_AmbigOverride`) is produced at the point of the override in that case. We have plenty of tests for that error, and in any case it isn't affected by this PR.

---
In reply to: [441903992](https://github.com/dotnet/roslyn/pull/44025#discussion_r441903992) [](ancestors = 441903992,441902428)

Reply:
> This warning was not used for that scenario. A different error (CS0462 ERR_AmbigOverride) is produced at the point of the override in that case.

What about the scenario in #44067? The removal of this warning is justified by that issue in ErrorCode.cs.

---
In reply to: [441923177](https://github.com/dotnet/roslyn/pull/44025#discussion_r441923177) [](ancestors = 441923177,441903992,441902428)

Reply:
Please add a test for that scenario with a different order of methods in base.

---
In reply to: [441960774](https://github.com/dotnet/roslyn/pull/44025#discussion_r441960774) [](ancestors = 441960774,441923177,441903992,441902428)